server/audit_math/sampler.py

```diff
@@ -121,16 +121,19 @@ def draw_ppeb_sample(
 
     # Map each batch to its weighted probability of being picked
     weighted_errors = [
-        macro.compute_max_error(batch_results[batch], contest) / U
+        float(macro.compute_max_error(batch_results[batch], contest) / U)
         for batch in batch_results
     ]
 
-    sample: List[Tuple[Any, Any]] = (
-        generator.choice(
-            list(batch_results.keys()),
-            sample_size + num_sampled,
-            p=weighted_errors,
-            replace=True,
+    sample: List = (
```

Comment:
Same comment as above. I'm guessing something about typechecking changed with poetry?

Reply:
Yeah, I let numpy get upgraded to it's latest version, which slightly changed the typechecking for that library. I could just lock it back to the version we had it at before, but it seemed like a minor enough change to let it slide